src/components/DeleteButtonEdge.tsx

```diff
@@ -112,6 +114,13 @@ const DeleteButtonEdge = ({
           </Popover>
         </span>
       </foreignObject>
+      {label && (
+        <EdgeText
+          x={edgeCenterX + foreignObjectSize / 2}
```

Comment:
To avoid overlaying the button, moving the label away from foreign object. Is there a better way to layout it?

Reply:
This is not so different to how we used to space out the steps, so if there is a better way then I'm not aware. :D I do wonder how much we would be able to integrate ELK with the labels, right now it's 100% handled by React Flow. ELK has a concept of labels, but I'm not sure what those labels mean exactly and if it's the same thing: https://www.eclipse.org/elk/reference/options/org-eclipse-elk-labelManager.html